docs/source/standard/document.rst

```diff
@@ -17,28 +17,31 @@ Schema
 :documentType:
```

Comment:
:id:		 
     string, auto-generated, read-only

Internal identifier of the object within an array.